libgnucash/engine/kvp-frame.hpp

```diff
@@ -204,11 +204,15 @@ struct KvpFrameImpl
      * @return The value at the key or nullptr.
      */
     KvpValue* get_slot(Path keys) const noexcept;
-    /** Convenience wrapper for std::for_each, which should be preferred.
+
+    void for_each_slot(void (*)(char const *key, KvpValue*, void*data), void*data) const noexcept;
```

Comment:
As far as I can understand you declare this function but removed its definition (in kvp-frame.cpp). Yet the function is still called in qofinstance-p.h:187
I'm surprised this even compiles, but perhaps you can explain ?

Reply:
I lost track of this one. Let me see what I can see.

Reply:
Good ol' C++ (probably C, too): only pay for what you use. It's called in `qofinstance-p.h`:187 as you say, but since *that* function (`qof_instance_foreach_slot_temp`) is never called, the definition is never needed. Thank you for finding this. I'll commit a proper removal presently.